diagnostic.md

```diff
@@ -16,23 +16,24 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Github is an online database that allows programmers to update changes they've made to a file, locally, on the web. It allows developers to keep track of each and every individual change they make to a file, as a means of record.
 
 ## Question 3
 
 ```js
 (false || true) && (false && true);
+
 ```
 
 Is the value of this expression `true` or `false`?  Explain your answer.
 
-_Replace this text with your answer._
+True. First half equates to True because  || defaults to true if either side is True.  Not 100% confident on how && works, so I will stick with True.
 
```

Comment:
You got the left side of the statement right, but the right side evaluates to false because, for two statements A and B, the expression A && B evaluates to true only when both A and B are true.  Here, false && true evaluates to false, which reduces the entire expression to true && false, which evaluates to false.